cypress/integration/AutoTest.js

```diff
@@ -18,5 +25,26 @@ describe('AutoTest Project', () => {
         // After search entered into search bar there should be on a new URL which includes '/commercial-ice-machines'
         cy.url().should('include', '/commercial-ice-machines')
 
+
+          // Tests vendor filter function for "Manitowoc" and verifies product count in vendor filter matches count in product list
+        
+
+        // Searches webpage for "Full-Dice Ice Machines" link. Clicks link and checks new webpage for proper address
+        cy.get('.item-categories').contains('Full-Dice Ice Machines').click()
+        cy.url().should('include', '/full-dice')
+
+        // Clicks the "Manitowoc" vendor checkbox using the Manitowoc brand value 
+        cy.get('.js-filter-value').check('09e51896-5085-42fc-9265-c23b5520db07')
```

Comment:
It is good practice do not use unclear or strange values like '09e51896-5085-42fc-9265-c23b5520db07'. This checkbox can be found and clicked by `data-title="Manitowoc"`.